backend/src/server/routes/v1/identity-ldap-auth-router.ts

```diff
@@ -202,10 +202,11 @@ export const registerIdentityLdapAuthRouter = async (server: FastifyZodProvider)
       }),
       body: z
         .object({
-          url: z.string().trim().min(1).describe(LDAP_AUTH.ATTACH.url),
-          bindDN: z.string().trim().min(1).describe(LDAP_AUTH.ATTACH.bindDN),
-          bindPass: z.string().trim().min(1).describe(LDAP_AUTH.ATTACH.bindPass),
-          searchBase: z.string().trim().min(1).describe(LDAP_AUTH.ATTACH.searchBase),
+          templateId: z.string().trim().optional().describe(LDAP_AUTH.ATTACH.templateId),
+          url: z.string().trim().optional().describe(LDAP_AUTH.ATTACH.url),
+          bindDN: z.string().trim().optional().describe(LDAP_AUTH.ATTACH.bindDN),
+          bindPass: z.string().trim().optional().describe(LDAP_AUTH.ATTACH.bindPass),
+          searchBase: z.string().trim().optional().describe(LDAP_AUTH.ATTACH.searchBase),
```

Comment:
I wonder if we can use the `discriminatedUnion` magic here instead so that it appears as two different schemas in the API spec